packages/govern-server/src/core/data/index.ts

```diff
@@ -54,78 +53,90 @@ class ConnectorTheGraph {
     })
   }
 
-  private async fetchResult<T>(
+  private async fetchResult<ReturnType>(
```

Comment:
Cause ``fetchResult`` is anyways a ``async`` method should we remove the ``callback`` which is a legacy pattern and just use the async/await construct :-)